extensions/objects/templates/image_with_regions_editor.html

```diff
@@ -80,3 +78,15 @@ <h4 class="modal-title">
     <button type="button" ng-click="confirmClear()" class="btn btn-danger">Clear Image and Regions</button>
   </div>
 </script>
+<style type='text/css'>
+  input.region-label-editor {
```

Comment:
I copied from the class on the svg item, to add the "oppia-image-with-regions-" prefix, so the full name is now "oppia-image-with-regions-region-label-editor".